pages/wiki/creating-an-asteroid-app.md

```diff
@@ -152,10 +151,88 @@ Change the following `Environment` variables:
 
 Your app should now be able to run from the application when you click the start button in the bottom left sidebar.
 
+# Learn QML
+
+Main docs: https://doc.qt.io/ (AOS uses qt5)
+
+AOS specific component/API: https://github.com/AsteroidOS/qml-asteroid
```

Comment:
Here too, a sentence would help. Eg:

qml-asteroid (link) provides APIs and UI components specific to AsteroidOS